packages/app-project/src/screens/ClassifyPage/components/RecentSubjects/RecentSubjects.js

```diff
@@ -42,10 +41,7 @@ function RecentSubjects({
   const placeholders = [...Array(size - displayedRecents.length)]
 
   return (
-    <ContentBox title={t('Classify.RecentSubjects.title', { projectName })}>
-      <Paragraph margin={{ top: 'none' }}>
-        {t('Classify.RecentSubjects.text')}
-      </Paragraph>
+    <ContentBox title={t('Classify.RecentSubjects.title')}>
```

Comment:
Removal of the Paragraph is per designer request, and the `projectName` variable was never actually used anywhere in this component. Removed it everywhere else related to RecentSubjects too.